string property
string obj1
string obj2
---
bool result_check
